.github/workflows/update-ami.py

```diff
@@ -1,51 +1,57 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 import json
 import os 
 import boto3
 import uuid
 import re
-import packer
+import subprocess
 
-uid = str(uuid.uuid4())
-file = '.github/workflows/amichange.json'
-packerfile = ".github/packer/ubuntu-focal-x86_64-public-ami.json"
+# Globals
+amifile = '.github/workflows/amichange.json'
+packerfile = ".github/packer/ubuntu-jammy-x86_64-public-ami.json"
 
-update_marketplace = True
+# Environment Globals
 product_id = os.getenv('PRODUCT_ID')
 role_arn = os.getenv('ROLE_ARN')
 vtag = os.getenv('TAG')
 tag = vtag.replace('v', '')
-variables = [product_id,role_arn,tag]
+skip_create_ami = os.getenv('SKIP_CREATE_AMI', True)
```

Comment:
We use `"True"` below, so we need to stay consistent and using the string here as the default rather than the actual `True` value.
```suggestion
skip_create_ami = os.getenv('SKIP_CREATE_AMI', 'True')
```

Reply:
Good call,  totally missed that.